BaseTools/Conf/build_rule.template

```diff
@@ -711,3 +753,39 @@
         "$(RC)" $(RC_FLAGS) $(OUTPUT_DIR)(+)$(MODULE_NAME)hii.rc ${dst}
     <Command.XCODE>
         "$(GENFW)" -o $(OUTPUT_DIR)(+)$(MODULE_NAME)hii.rc -g $(MODULE_GUID) --hiibinpackage $(HII_BINARY_PACKAGES) $(GENFW_FLAGS)
+
+# MU_CHANGE Start

```

Comment:
This section is not part of the OemData/ProductDataFormatter changes. It is from this commit https://github.com/microsoft/mu_basecore/commit/b77ef3e5c158676adbe7d95b5db8a2b3b3f6394a.

Unfortunately, it looks like that commit has been conflict resolved over time to pick up some OEM data formatter changes but that should be separated out now.

Reply:
I removed any reference to that commit which is labelled to be dropped.